local_planner/src/nodes/local_planner.cpp

```diff
@@ -404,7 +401,10 @@ void LocalPlanner::updateObstacleDistanceMsg() {
 void LocalPlanner::reprojectPoints(Histogram histogram) {
   double n_points = 0;
   double dist, age;
+  float half_res = ALPHA_RES / 2;
   Eigen::Vector3f temp_array[4];
+
+  std::vector<PolarPoint> p_pol(4);
```

Comment:
Rather make this a matrix than a vector, since you know beforehand how many you actually need. Or better yet you can use a std::array